content/installation/upgrading_go.md

```diff
@@ -58,7 +58,7 @@ sudo apt-get install go-server
 
 Run the GoCD installer as described
 ```bash
-sudo rpm -Uvh /path/to/go-server-${version}.deb
+sudo rpm -i /path/to/go-server-${version}.rpm
```

Comment:
The extension should be corrected, but since this is about upgrades, isn't `-Uvh` correct? `-i` is for installation, right? Or does it work for both?

I think there should ls be a line like `# OR` in between the two commands, as in the Debian example.

Reply:
Updated the command based on comment